pkg/deployment/deployment_project.go

```diff
@@ -119,7 +120,13 @@ func (p *DeploymentProject) processConfig() error {
 		if len(item.Tags) != 0 {
 			continue
 		}
+
 		if item.Path != nil {
+			// Validate or fix item.Path, as k8s labels has constrains
+			// regex used for validation is '(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?')
+			if *item.Path == "." {
```

Comment:
This change make this possible ((path: .) & highlight that for example `_stage0` as invalid etc..

```
deployments:
  - path: .
  - git:
      url: git@gitlab.com:
```

Reply:
Can you put this into a separate PR so that it properly appears in the changelog later?